gitbr.go

```diff
@@ -20,6 +21,15 @@ type UIRunner interface {
 
 // Open returns an UIRunner from a git repository filesystem path.
 func Open(path string) (UIRunner, error) {
+	_, err := os.Stat(path + "/.git")
+	if err != nil && !os.IsNotExist(err) {
+		return nil, err
+	}
+
+	if os.IsNotExist(err) {
+		return nil, fmt.Errorf("no .git directory detected (needs to be in repository toplevel).")
+	}
```

Comment:
I think this is already covered by https://github.com/gonzaloserrano/git-br/blob/858332d5992a11ef09df768bacd66e1e4249583f/vendor/gopkg.in/src-d/go-git.v4/repository.go#L221-L222 in the `PlainOpen` call so we don't need to do it twice.